src/app/qgsmaptooladdcircularstring.cpp

```diff
@@ -83,6 +86,16 @@ void QgsMapToolAddCircularString::keyPressEvent( QKeyEvent* e )
 
     createCenterPointRubberBand();
   }
+
+  if ( e && e->key() == Qt::Key_Escape )
+  {
+    mPoints.clear();
+    delete mRubberBand; mRubberBand = nullptr;
```

Comment:
I see that there are other occurrences of this style. I think having each command on it's own line makes code much more readable and today's screens fortunately are big enough to show a couple of lines so we don't need to save vertical space ;)